client/plots/hierCluster.interactivity.js

```diff
@@ -203,6 +203,35 @@ export function setClusteringBtn(holder, callback) {
 						}*/
 					]
 				},
+				{
+					label: `Distance Method`,
+					title: `Sets which clustering method to use`,
+					type: 'radio',
+					chartType: 'hierCluster',
```

Comment:
at plot init, values coded in plot state should be validated against this array

Reply:
I added if statements in hierCluster.js line 184 & 185.